dev/Kozani/KozaniHostRuntime/main.cpp

```diff
@@ -112,60 +115,68 @@ void LogArgs(int argc, PWSTR* argv)
     }
 }
 
-std::wstring GetRemoteAumidFromCommandLineArgs(PCWSTR commandLine)
+// Remote app AUMID is stored in an app extension of the local variant package (LVP). The app extension name is formatted as:
+//     com.microsoft.kozani.localvariant.remoteapp.<Converted LVP package family name>
+// and the app extension Id is the package-relative application identifier (PRAID) of the LVP app being activated.
+// <Converted LVP package family name>: we replace the '_' character in the package family name with '-' as '_' is not allowed in the app extension name.
+// 
+// For example:
+// <uap3:Extension Category="windows.appExtension">
+//   <uap3:AppExtension Name = "com.microsoft.kozani.localvariant.remoteapp.Calculator-fy70fzsv57rkp"
+//     Id = "App"
+//     DisplayName = "Remote app AUMID"
+//     Description = "The App User Mode ID of the remote app this local variant app represents">
+//     <uap3:Properties>
+//       <RemoteAppAumid>Microsoft.WindowsCalculator_8wekyb3d8bbwe!App</RemoteAppAumid>
+//     </uap3:Properties>
+//   </uap3:AppExtension>
+// </uap3:Extension>
+winrt::hstring GetRemoteAumid()
 {
-    int argc{};
-    const wil::unique_hlocal_ptr<PWSTR[]> argv{ CommandLineToArgvW(commandLine, &argc) };
-    LogArgs(argc, argv.get());
-
-    for (int i = 0; i < argc; i++)
+    WCHAR lvpAumid[APPLICATION_USER_MODEL_ID_MAX_LENGTH + 1]{};
+    UINT32 lvpAumidLength{ ARRAYSIZE(lvpAumid) };
+    THROW_IF_WIN32_ERROR(GetCurrentApplicationUserModelId(&lvpAumidLength, lvpAumid));
+
+    WCHAR lvpFamilyName[PACKAGE_FAMILY_NAME_MAX_LENGTH + 1]{};
+    WCHAR lvpRelativeId[PACKAGE_RELATIVE_APPLICATION_ID_MAX_LENGTH + 1]{};
+    UINT32 lvpFamilyNameLength{ ARRAYSIZE(lvpFamilyName) };
+    UINT32 lvpRelativeIdLength{ ARRAYSIZE(lvpRelativeId) };
```

Comment:
SUGGESTION: `lvpPraid`

You use `Aumid` below but not `Praid` here

Reply:
Changed.